content/ember-released-5-9.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Ember 5.9 Released
+authors:
+  - jared-galanis
+date: 2024-06-06T00:00:00.000Z
+tags:
+  - releases
+  - '2024'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.9 of Ember.js and Ember CLI. This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.
```

Comment:
5.9 isn't an LTS candidate ... this is the point 5.8 becomes an LTS though

Reply:
Ah thanks, good catch. I had copied the 5.8 post. 😅